ansible/run-perf.yml

```diff
@@ -28,7 +28,7 @@
       when: (prometheus_logging | bool)
 
 - name: entitle host and manage repos
-  hosts: microshift
+  hosts: microshift-dev
```

Comment:
`microshift` is a host group, `microshift-dev` is a single host. We don't want to change it to `microshift-dev`.